styles/global.css

```diff
@@ -3,3 +3,14 @@ body,
 #__next {
   height: 100%;
 }
+
```

Comment:
Added some CSS since some of these styles aren't easily doable in bootstrap afaik. Lemme know if you want me to move these to the jsx CSS section in the component file, just figured I'd put it here so that vscode can lint it.